.github/cspell.json

```diff
@@ -3,43 +3,186 @@
+    "fullscreen",
+    "gamepads",
+    "hitboxes",
+    "hoverable",
+    "hoverables",
+    "ints",
+    "Kawabunga",
+    "kinect",
+    "klingsbo",
+    "LTRB",
+    "LTRBR",
+    "LTWH",
+    "Lukas",
+    "multitap",
+    "mutators",
+    "NPCs",
+    "Nakama",
+    "padracing",
+    "patreon",
+    "queryable",
+    "RGBO",
+    "raycasting",
+    "rects",
+    "respawned",
+    "rgba",
+    "SVGs",
+    "schnurber",
+    "skia",
+    "strikethrough",
+    "tappable",
+    "tappables",
+    "Tavian",
+    "tiled's",
+    "viewport's",
+    "WASD",
+    "WBMP",
+    "widget's",
+    "abelian",
+    "ambiguate",
+    "antialiasing",
+    "artboard",
+    "asprite",
+    "autofocus",
     "backtick",
     "backticks",
+    "ballons",
+    "behaviour",
     "blockquote",
+    "bloodlust",
     "broadphase",
+    "cathetus",
+    "caveace",
+    "codepoint",
+    "codepoints",
+    "collidables",
+    "colourful",
     "conformant",
     "dartdoc",
     "dartdocs",
     "dashbook",
     "discoverability",
+    "draggable",
+    "draggables",
+    "erayzesen",
+    "erickzanardo",
+    "euclidian",
+    "everytime",
+    "feroult",
     "fireslime",
     "flippable",
+    "fullscreen",
+    "gameover",
+    "gamepads",
+    "gameplay",
+    "gapless",
+    "glyph's",
+    "goldens",
+    "grayscale",
+    "greyscale",
+    "highscore",
     "hitbox",
+    "hitboxes",
+    "janky",
+    "lerp",
+    "lexer's",
     "linkcheck",
-    "LTRBR",
-    "LTWH",
+    "luanpotter",
     "markdownlint",
+    "microtask",
+    "mixins",
+    "monospace",
+    "mutators",
+    "netcode",
+    "overridable",
+    "padracing",
+    "perfoming",
+    "performant",
+    "pgainullin",
     "platformer",
+    "polyline",
     "positionable",
+    "postmultiply",
+    "preorder",
+    "prepartion",
     "prerendered",
     "pressable",
     "proxying",
+    "ptero",
+    "pubdev",
+    "pubspec",
     "raycast",
+    "raycasting",
+    "raycasts",
     "raytracing",
+    "rebalance",
+    "rects",
     "refactor",
     "refactoring",
     "refactorings",
+    "reimplement",
+    "reimplemented",
     "renderable",
+    "respawn",
+    "respawned",
+    "rrect",
+    "rsts",
+    "scos",
+    "serializable",
     "socio",
-    "Strikethrough",
+    "spritecow",
+    "spritesheet",
+    "spydon",
+    "ssin",
     "stylesheet",
+    "subfolders",
+    "sublist",
     "sublists",
-    "typedef",
-    "typedefs",
+    "svgs",
+    "tappable",
+    "tilemap",
+    "tileset",
+    "tilesets",
     "toctree",
     "toctrees",
-    "vsync"
+    "trex",
+    "tweening",
+    "typedef",
+    "typedefs",
+    "unfollow",
+    "unmount",
+    "unproject",
+    "unrenderable",
+    "unscale",
+    "updatesystem",
+    "viewport's",
+    "viewports",
+    "vsync",
+    "wbmp",
+    "webp",
+    "wolfenrain"
   ],
   "ignorePaths": ["**/media/**", "**/assets/**", "**/test/**"],
   "ignoreRegExpList": ["@\\w+", "\\$\\w+", "\\{\\w+", "\\`\\w+", "\\:\\w+", "\\/\\w+", "\\#\\w+"],
```

Comment:
`"@\\w+"` -> `"@[\\w\\-]+"`, since GitHub names can contain dashes in them
`"@\\w+"` -> `"@[\\w\\-]+"`, for URLs